rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -207,6 +209,7 @@ def __init__(
         self.rpc_timeout = rpc_timeout
         self.chain_id: SUPPORTED_CHAIN_IDS = blockchain.to_chain_id()  # type: ignore[assignment]
         self.chain_name = self.blockchain.name.lower()
+        self.native_asset = native_asset
```

Comment:
Here and elsewhere please be consistent. Call it token. In the same file you end up using different terms for the same thing.

```suggestion
        self.native_token = native_asset
```